components/HeaderedControls/samples/HeaderedTreeViewSample.xaml.cs

```diff
@@ -0,0 +1,108 @@
+    {
+        var list = new ObservableCollection<ExplorerItem>();
+        ExplorerItem folder1 = new ExplorerItem()
+        {
+            Name = "Work Documents",
+            Children =
+                {
+                    new ExplorerItem()
+                    {
+                        Name = "Functional Specifications",
+                        Children =
+                        {
+                            new ExplorerItem()
+                            {
+                                Name = "TreeView spec",
+                              }
+                        }
+                    },
+                    new ExplorerItem()
+                    {
+                        Name = "Feature Schedule",
+                    },
+                    new ExplorerItem()
+                    {
+                        Name = "Overall Project Plan",
+                    },
+                    new ExplorerItem()
+                    {
+                        Name = "Feature Resources Allocation",
+                    }
+                }
+        };
+        ExplorerItem folder2 = new ExplorerItem()
+        {
+            Name = "Personal Folder",
+            Children =
+                        {
+                            new ExplorerItem()
+                            {
+                                Name = "Home Remodel Folder",
+                                Children =
+                                {
+                                    new ExplorerItem()
+                                    {
+                                        Name = "Contractor Contact Info",
+                                    },
+                                    new ExplorerItem()
+                                    {
+                                        Name = "Paint Color Scheme",
+                                    },
+                                    new ExplorerItem()
+                                    {
+                                        Name = "Flooring Woodgrain type",
+                                    },
+                                    new ExplorerItem()
+                                    {
+                                        Name = "Kitchen Cabinet Style",
+                                    }
+                                }
+                            }
+                        }
+        };
+
+        list.Add(folder1);
+        list.Add(folder2);
+        return list;
+    }
+}
+    public class ExplorerItem
+    {
+        public string? Name { get; set; }
+    private ObservableCollection<ExplorerItem>? m_children;
+    public ObservableCollection<ExplorerItem> Children
+    {
+        get
+        {
+            if (m_children == null)
+            {
+                m_children = new ObservableCollection<ExplorerItem>();
+            }
+            return m_children;
+        }
+        set
+        {
+            m_children = value;
+        }
+    }
+
+}
```

Comment:
Brace formatting is weird here. +Private variable is just `_children`? https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case

Thought we had editorconfig/style warnings for this stuff...? @Arlodotexe?

Reply:
Sloppy copy-and-paste from the WinUI Gallery sample :). Fixed now!